src/chapters/06/main.tex

```diff
@@ -25,44 +25,66 @@ \section{Problem}\label{sec:problem}
         profit.
 \end{itemize}
 
-The expected profits are given in Table~\ref{tbl:profit-of-taxi-companies}.
+The expected profits for Taxi Company A are given in Table~\ref{tbl:profit-of-taxi-companies}.
+The profits for Taxi Company B are the same.
 
+\newcolumntype{P}[1]{>{\centering\arraybackslash}p{#1}}
 \begin{table}[!hbtp]
     \begin{center}
-    \begin{tabular}{l|ccc}
-        \diagbox{Taxi numbers}{Other company taxi numbers} & 1     & 2      & 3\\
+    \begin{tabular}{cl|P{0.08\textwidth}P{0.08\textwidth}P{0.08\textwidth}}
+                                                                  &    & \multicolumn{3}{c}{Taxi Company B}\\
+                                                                  &    & 1               & 2                 & 3\\
         \toprule
-        1                                                  & 1     & \(\frac{1}{2}\) & \(\frac{1}{3}\)\\
-        2                                                  & \(\frac{3}{2}\)   & \(\frac{19}{20}\)& \(\frac{1}{2}\)\\
-        3                                                  & \(\frac{5}{3}\) & \(\frac{4}{5}\)  & \(\frac{17}{20}\)\\
+        \multirow{3}{*}{\rotatebox[origin=c]{90}{Taxi Company A}} & 1  & 1               & \(\frac{1}{2}\)   & \(\frac{1}{3}\)\\[4.5mm]
+                                                                  & 2  & \(\frac{3}{2}\) & \(\frac{19}{20}\) & \(\frac{1}{2}\)\\[4.5mm]
+                                                                  & 3  & \(\frac{5}{3}\) & \(\frac{4}{5}\)   & \(\frac{17}{20}\)\\[4.5mm]
```

Comment:
I think the way it was is better, let's chat about this on Friday.

Reply:
We have agreed to duplicate the table, this means we can remove a sentence from the test.